### ISSUE TITLE
MacOS ARM `.whl` 

### ISSUE THREAD

**Issue by BradyAJohnston** [biopython/biopython#4315]
Is it possible to have pre-built `.whl` available for ARM MacOS via PyPi?

I'm the developer of [Molecular Nodes](https://github.com/BradyAJohnston/MolecularNodes) which is an add-on for the 3D modelling and animation program Blender. The add-on depends on installation of a bunch of python packages inside of Blender's bundled python, which do all of the heavy lifting in terms of file parsing etc. Blender does not ship with the required files to compile new packages inside the Bundled python, and so it requires pre-built `.whl` files to be available for to be able to install any new packages.

One of the major dependencies is [MDAnalysis](https://github.com/MDAnalysis/mdanalysis) which in turn depends on `biopython`. On M1 / M2 macs, installation fails as there currently isn't pre-built `.whl`s for those platforms. Would it be possible to potentially add this to the PyPi page?

Currenlty the solution for users is to [pre-compile their own required packages](https://github.com/BradyAJohnston/MolecularNodes/issues/108) and install from there locally, but this complicates the installation process for new less tech-savvy users so I'm hoping to get it down to a single-button install for all platforms (currently just M1 / M2 MacOS that it fails on).

**Comment by JoaoRodrigues** [comment]
Hi Brady,

There are M1 runners on GHA so that could definitely be an option. We're
planning a release in the coming days so I'll add this to our to-do list.

Cheers,

João

A domingo, 28/05/2023, 20:09, Brady Johnston ***@***.***>
escreveu:

**Comment by peterjc** [comment]
Yes, should be possible now: https://github.com/pypa/cibuildwheel

**Comment by BradyAJohnston** [comment]
Thanks for the feedback, that would be amazing if possible! Appreciate it and looking forward to the release.

**Comment by peterjc** [comment]
According to https://cibuildwheel.readthedocs.io/en/stable/faq/#apple-silicon 

> Only CPython 3.8 and newer support universal2 and arm64 wheels.

We've dropped Python 3.7 support as part of Biopthyon 1.82 anyway.
